template/cordova/lib/prepare.js

```diff
@@ -40,14 +40,37 @@ module.exports.applyPlatformConfig = function () {
         return false;
     });
 
+    // Apply appxmanifest changes
     ['package.windows.appxmanifest', 'package.windows80.appxmanifest', 'package.phone.appxmanifest'].forEach(
         function(manifestFile) {
             updateManifestFile(config, path.join(ROOT, manifestFile));
     });
 
+    // Apply jsproj changes
+    ['CordovaApp.Phone.jsproj', 'CordovaApp.Windows.jsproj', 'CordovaApp.Windows80.jsproj'].forEach(
+        function(jsprojFile) {
+            updatejsprojFile(config, path.join(ROOT, jsprojFile));
+    });
+
     copyImages(config);
 };
 
+function updatejsprojFile(config, jsProjFilePath) {
+    var defaultLocale = config.defaultLocale();
+
+    if (defaultLocale) {
```

Comment:
I think we should remove or set to default DefaultLanguage in case there is no config.defaultLocale().